server.rb

```diff
@@ -1,25 +1,9 @@
 require 'sinatra'
 require 'sinatra/streaming'
-require_relative 'render'
 
 set :server, 'puma'
 set :port, (ENV['PORT'] || 8580).to_i
 
-get '/update' do
```

Comment:
Although I'm not sure how this endpoint is called frequently, should it be left as empty method until stop calling? (If the cron like job is already stopped, never mind.)

Reply:
The cron job has been stopped, and actually the past cron job was not via HTTP. (Calling `render.rb` directly from Heroku's feature.)